src/pages/ProfilePage.js

```diff
@@ -95,7 +95,7 @@ const ProfileInfo = styled.h2`
   font-size: 23px;
 `;
 
-const Header = styled.h3`
+const PageNav = styled.h3`
```

Comment:
If you name the component "PageNav" it is better to use `nav` as the html-element, otherwise it is a bit irritating...
I think it is fine if you use a `nav`Element 🤔